### PR TITLE
Removed unnecessary request object cloning when cumulatively caching all new requests 

### PR DESCRIPTION
**Pull request by JasonRammoray**
What's changed, or what was fixed?
- simplified a code snippet demonstrating a cumulative request caching in [Cache and return requests](https://developers.google.com/web/fundamentals/primers/service-workers/#cache_and_return_requests) section on [Service Workers: an Introduction](https://developers.google.com/web/fundamentals/primers/service-workers) page.

**Fixes:** #7064 

**Target Live Date:** 2019-01-09

- [x] This has been reviewed and approved by (petele)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by JasonRammoray** [comment]
@petele , could you, please, take a look at that when you have time?
May I kindly ask you to put a respective label on the pull-request since I don't have enough permissions?
Thanks in advance.

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,12 @@ tools/claat
 gcloud-client-secret.json
 package-lock.json
 
+## IDE related files
```
Comment (petele) [inline .gitignore]: Can you remove this file from the PR please?

You can add IDE related files to your personal `.gitignore` file, and `index.yaml` files are not generated anywhere, there are some `_index.yaml`, but those are already handled.

File: src/content/en/fundamentals/primers/service-workers/index.md [hunk 2]
```diff
@@ -273,13 +273,7 @@ response of the fetch request and then adding it to the cache, like below.
               return response;
             }
 
-            // IMPORTANT: Clone the request. A request is a stream and
```
Comment (petele) [inline src/content/en/fundamentals/primers/service-workers/index.md]: @jakearchibald - can you confirm this and either request changes or approve the PR?

File: .gitignore (hunk 1 above)
Comment (JasonRammoray) [reply .gitignore]: Sure.
I guess [that could explain](https://cloud.google.com/appengine/docs/standard/python/config/indexconfig#creating_indexes_using_the_development_server) why I had autogenerated index.yaml upon the first launch on my machine.

Done.

File: src/content/en/fundamentals/primers/service-workers/index.md (hunk 2 above)
Comment (JasonRammoray) [reply src/content/en/fundamentals/primers/service-workers/index.md]: Jake has confirmed the changes are correct.

**Comment by JasonRammoray** [comment]
@petele, all changes have been incorporated.
Would you mind merging this one, please?

**Pull request merged**